CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+1.0.0 (March 2018)
+------------------
+
+This version includes:
+
+
+- handled correctly data access depending on permissions level
+- added new icon for polyline
+- added minimal support for masks
+- improved UI
+- handled images with large number of channels
+- added support to open the image in other applications installed e.g. OMERO.figure
```

Comment:
to open the image in other applications installed -> for opening images in other installed applications